reaper/manager.go

```diff
@@ -395,11 +395,10 @@ func (r *BucketReaper) busy(bucket string) (busy bool, err error) {
 //
 // Delete bucket.
 func (r *BucketReaper) delete(path string) (err error) {
-	cmd := exec.Command("/usr/bin/rm", "-rf", path)
-	b, err := cmd.CombinedOutput()
+	err = nas.RmDir(path)
```

Comment:
Using nas.RmDir() because it's 12x faster than os.RemoveAll() which is really slow.